cfbs.json

```
@@ -699,6 +700,24 @@
         "copy ./sudo-requires-password.cf services/autorun/sudo-requires-password.cf"
       ]
     },
+    "surf-cfengine-library": {
+      "description": "SURF CFEngine Library (SCL) for building services with json/mustache",
+      "tags": ["inventory", "lib", "management", "experimental"],
+      "repo": "https://github.com/basvandervlies/cf_surfsara_lib",
+      "by": "https://github.com/basvandervlies/",
+      "version": "1.1.0",
```

@basvandervlies I understand that the version number of SCL is 1.1.0, but I will set it to version 0.1.0 in CFEngine Build, _for now_. This is because currently it is a little difficult to keep the version numbers in sync if we want to make changes. For example, if we want to make a change to the SCL module in CFEngine Build, we would have to release it as 1.1.1,  but you might not want to release a new version just because of some very minor change we did.

```suggestion
      "version": "0.1.0",
```

Sorry for the inconvenience, we have the same problem with masterfiles, and we will work on enabling `-2` and `-3` version releases to help with the situation.

https://tracker.mender.io/browse/CFE-3905